.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
```

Comment:
I've moved the links from the 'Help request' template here so as to encourage help requests to go in the appropriate forums as opposed to in GitHub where they are usually ignored.

Reply:
What does it do?

Reply:
It's documented here:

https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Basically, it sets up links which appear on this page:

https://github.com/WordPress/gutenberg/issues/new/choose

Here's an example of how it looks:

https://github.com/Homebrew/brew/issues/new/choose

Reply:
Ah yeah, saw the docs, but it didn't really show what it does. This is cool, makes those links much more obvious 👍